src/test/java/nl/rijksoverheid/mev/gezagsmodule/service/ToelichtingServiceTest.java

```diff
@@ -0,0 +1,161 @@
+package nl.rijksoverheid.mev.gezagsmodule.service;
+
+import nl.rijksoverheid.mev.gezagsmodule.domain.VeldenInOnderzoek;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ToelichtingServiceTest {
+
+    private ToelichtingService classUnderTest;
+
+    private static final String BASE_TOELICHTING = "mock";
+    private static final String BASE_MISSENDE_VELDEN_TOELICHTING = "missende gegevens: %s";
+    private static final String FIELD_1 = "field 1";
+    private static final String FIELD_2 = "field 2";
+    private static final String IN_ONDERZOEK_NEEDLE = "waardes in onderzoek waren gedetecteerd";
+    private static final String IN_ONDERZOEK_PERSOONSVELDEN = "Persoonsvelden";
+    private static final String IN_ONDERZOEK_OUDER_1 = "ouder 1";
+
+    @BeforeEach
+    public void ToelichtingServiceTest() {
```

Comment:
Dit is geen constructor is heeft wel de naming convention van een constructor; dat is verwarrend. Vaak worden `@BeforeEach`-annotated methods `setup()` genoemd.